star_tides/services/databases/mongo/models/guidance_model.py

```diff
@@ -0,0 +1,15 @@
+''' star_tides.services.databases.mongo.models.guidance_model
+'''
+from mongoengine import EmbeddedDocument, StringField, ReferenceField
+from star_tides.services.databases.mongo.models.contact_model import (
+    ContactModel
+)
+
+
+class GuidanceModel(EmbeddedDocument):
+    contact = ReferenceField(ContactModel, required=False)
```

Comment:
Nit: call this `author` instead of `contact`? Or add a comment saying something like `this is the author of the guide`.

Reply:
I've been using `ObjectIdField` since it doesn't have any clever linking logic.